src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -128,6 +132,21 @@ public void updateFilteredPersonList(Predicate<Person> predicate) {
         filteredPersons.setPredicate(predicate);
     }
 
+    //=========== Ui Changing =============================================================
+
+    public void linkUi(Ui ui) {
+        this.ui = ui;
+    }
+
+    @Override
+    public void showPerson(Person person) {
+        requireNonNull(person);
+        if (ui != null) {
```

Comment:
What is the expected behavior (or exception) when ui is null? Or could I get assured that ui cannot not be null (and hence do not need to go through this conditional branch?)

Reply:
This is a minor enhancement suggestion

Reply:
Thanks for the suggestion. The ui should not be null as it will be linked to the Model via the linkUi method in the MainApp.